libraries/llvm/buffer.c

```diff
@@ -13,127 +14,173 @@
 // with clang-generated LLVM from this file's entire translation unit.
 
 
-// Eight bytes for the reference counter.
+// - 8 bytes for the reference counter.
 // E.g. capacity could later also be stored in the header.
 #define BUFFER_HEADER_WIDTH (8)
```

Comment:
@phischu we still need to turn buffers into valid Pos, can we do this after merging this PR?